EIPS/eip-908.md

```diff
@@ -3,7 +3,7 @@ eip: 908
 title: Reward clients for a sustainable network
 author: James Ray (@jamesray1), Micah Zoltu (@MicahZoltu)
 discussions-to: https://ethereum-magicians.org/t/eip-908-reward-full-nodes-and-clients/241
-status: Draft
+status: Withdrawn
```

Comment:
```suggestion
status: Abandoned
```

So that this can be merged now.

Reply:
I suppose continuing to wait on this is just clogging things up at this point.  I capitulate.